src/common.py

```diff
@@ -118,10 +118,9 @@ def page_setup(page: str = "", help_text: str = "") -> dict[str, Any]:
             st.session_state.controlo = True
         else:
             st.session_state.location = "online"
-            st.session_state.controlo = False
-        # if we run the packaged windows version, we start within the Python directory -> need to change working directory to ..\umetaflow-gui-main
+        # if we run the packaged windows version, we start within the Python directory -> need to change working directory to ..\streamlit-template
         if "windows" in sys.argv:
-            os.chdir("../flashviewer")
+            os.chdir("../flashtaggerviewer")
```

Comment:
again, do we need to change this?

Reply:
I dont think this an urgent change. I will test it on my windows machine when I have time. But yes it should be renamed. I will put it on my ToDo list.